### DISCUSSION TITLE
esp32/Arduino/30.esp32-arduino-hc-sr04

### DISCUSSION THREAD

**Discussion by giscus** [bigrich-luo/DocsGiscus#31] [General]
# esp32/Arduino/30.esp32-arduino-hc-sr04

https://docs.geeksman.com/esp32/Arduino/30.esp32-arduino-hc-sr04.html

**Comment by zuowei176** [comment]
真的好喜欢up的课，啥时候在b站继续更新arduino基础的视频呀，真的好期待

**Comment by chensiluo** [comment]
delay的参数是毫秒, 为何你这里的代码就变成微秒了
